+bisos/@Iteration/run.m

```diff
@@ -59,6 +59,10 @@
     % current step
     step = obj.getstep(sidx);
     
+    if isprop(step,'data_prev') && step.data_prev
+       info.solutions = solution; 
+    end
+    
```

Comment:
That's better than your [previous suggestion](https://github.com/tcunis/bisosprob/issues/11#issuecomment-1132854761), but I still don't like it.

In another pull request (#14), I have added the possibility to store information for each step inside the info struct without risk of multiple steps accessing the same data. The proper way to make the previous solution accessible then would be to save, in the respective `bisos.iteration.step#run` method, the data which will be needed *in the next iteration*.